academy/static/src/js/dog.js

```diff
@@ -0,0 +1,20 @@
+odoo.define('academy.dog', function(require) {
+    'use strict';
+
+    var Animal = require('academy.animal');

```

Comment:
use camelcase for variables 
![image](https://github.com/AntonioMartinD/Building_a_website/assets/35231827/62ea50de-5505-439a-b85c-8fafbd95cb27)